### PR TITLE
Prevent usage from being printed when the start command encounters an error

### PR DESCRIPTION
**Pull request by dwedul-figure**
## Description

closes: #670

When an error is encountered during `provenanced start`:
* Before this change: the error is printed, then usage is printed, then the program exits.
* After this change: the error is printed, then the program exits.

You can still get usage info for the command using either `provenanced help start` or `provenanced start --help`.

---

Before we can merge this PR, please make sure that all the following items have been
checked off. If any of the checklist items are not applicable, please leave them but
write a little note why.

- [ ] Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/provenance-io/provenance/blob/main/CONTRIBUTING.md#pr-targeting))
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Wrote unit and integration [tests](https://github.com/provenance-io/provenance/blob/main/CONTRIBUTING.md#testing)
- [ ] Updated relevant documentation (`docs/`) or specification (`x/<module>/spec/`)
- [ ] Added relevant `godoc` [comments](https://blog.golang.org/godoc-documenting-go-code).
- [ ] Added a relevant changelog entry to the `Unreleased` section in `CHANGELOG.md`
- [ ] Re-reviewed `Files changed` in the Github PR explorer
- [ ] Review `Codecov Report` in the comment section below once CI passes

### CODE REVIEW

**Review by fkneeland-figure — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -69,6 +69,7 @@ Ref: https://keepachangelog.com/en/1.0.0/
   * `provenance.attribute.v1.MsgAddAttributeRequest` 10 hash (10,000,000,000 nhash)
   * `provenance.metadata.v1.MsgWriteScopeRequest`  10 hash (10,000,000,000 nhash)
   * `provenance.metadata.v1.MsgP8eMemorializeContractRequest` 10 hash (10,000,000,000 nhash)
+* When the `start` command encounteres an error, it no longer outputs command usage [#670](https://github.com/provenance-io/provenance/issues/670)
```
Comment (channa-figure) [inline CHANGELOG.md:72]: *encounters

**Review by channa-figure — CHANGES_REQUESTED** [review]

**Review by iramiller — APPROVED** [review]
Nice! ... is this a bug fix or an improvement :chinrub:

File: CHANGELOG.md (hunk 1 above)
Comment (dwedul-figure) [inline CHANGELOG.md:72]:
```suggestion
* When the `start` command encounters an error, it no longer outputs command usage [#670](https://github.com/provenance-io/provenance/issues/670)
```

Comment (dwedul-figure) [reply CHANGELOG.md]: Fixed! Thanks!

**Review by fkneeland-figure — DISMISSED** [review]

**Review by channa-figure — APPROVED** [review]

**Review by fkneeland-figure — APPROVED** [review]

**Pull request merged**